coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
You should probably add all files dealing with code coverage into your `.gitignore`